src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" PrivateAssets="All" />
-    <PackageReference Include="NUnit" Version="[3.12.0, 4.0.0)" />
```

Comment:
Aren't transport tests shipped as nuget?

Reply:
they are, why would we need a range though?

Reply:
Why did we add it in the first place?